js/google_ima.js

```diff
@@ -528,9 +530,39 @@ require("../html5-common/js/utils/utils.js");
        * @public
        * @method GoogleIMA#playAd
        * @param {object} ad The ad to play from the timeline.
+       * @param {object} adRequestOnly True to request the ad without starting playback, false to request and playback the ad
        */
-      this.playAd = function(amcAdPod)
+      this.playAd = function(amcAdPod, adRequestOnly)
       {
+        if (amcAdPod === null || typeof amcAdPod === 'undefined')
+        {
+          return;
+        }
+
+        if (!adRequestOnly)
+        {
+          _playAdOnRequestSuccess = amcAdPod;
```

Comment:
This sounds a bit like a boolean. Maybe we could rename it to something like `_adToPlayOnRequestSuccess`